open-api/rest-catalog-open-api.yaml

```diff
@@ -2033,6 +2537,68 @@ components:
         metadata-location:
           type: string
 
+    CreateViewRequest:
+      type: object
+      required:
+        - name
+        - schema
+        - view-version
+        - properties
+      properties:
+        name:
+          type: string
+        location:
+          type: string
+        schema:
+          $ref: '#/components/schemas/Schema'
+        view-version:
+          $ref: '#/components/schemas/ViewVersion'
+        properties:
+          type: object
+          additionalProperties:
+            type: string
+
+    LoadViewResult:
+      description: |
+        Result used when a view is successfully loaded.
+
+
+        The view metadata JSON is returned in the `metadata` field. The corresponding file location of view metadata is returned in the `metadata-location` field.
+        Clients can check whether metadata has changed by comparing metadata locations after the view has been created.
+
+
+        The `config` map returns view-specific configuration for the view's resources, including its HTTP client and FileIO.
+        For example, config may contain a specific FileIO implementation class for the view depending on its underlying storage.
+
+
+        The following configurations should be respected by clients:
+
+        ## General Configurations
+
+        - `token`: Authorization bearer token to use for view requests if OAuth2 security is enabled 
+
+        ## AWS Configurations
+
+        The following configurations should be respected when working with views stored in AWS S3
+         - `client.region`: region to configure client for making requests to AWS
+         - `s3.access-key-id`: id for for credentials that provide access to the data in S3
+         - `s3.secret-access-key`: secret for credentials that provide access to data in S3 
+         - `s3.session-token`: if present, this value should be used for as the session token 
+         - `s3.remote-signing-enabled`: if `true` remote signing should be performed as described in the `s3-signer-open-api.yaml` specification
```

Comment:
I don't think these AWS configurations are valid because we don't initialize a `FileIO` for the view when using the REST catalog.

Reply:
agreed, now that we don't need `FileIO` in the view ops class, we can remove these